functions/assignOfflineContact.ts

```diff
@@ -84,18 +105,13 @@ const assignToOfflineWorker = async (
     attributes: JSON.stringify({ ...previousAttributes, waitingOfflineContact: true }), // waitingOfflineContact is used to avoid other tasks to be assigned during this window of time (workflow rules)
   });
 
-  const result = await assignToAvailableWorker(event, newTask);
+  const result = await assignToAvailableWorker(event, newTask, 0);
```

Comment:
Consider giving `retry` a default value of `0`, rather than passing it in here.  However, not a big deal to me.